hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,175 @@
+defmodule GameState do
+  defstruct game_state: :initializing,
+            turns_left: 7,
+            letters: [],
+            used: [],
+            last_guess: ""
+end
+
+defmodule Hangman.Game do
+
+  # Create a new game state
+  def new_game() do
+    # Pick a random word from the dictionary
+    word = Dictionary.random_word()
+    |> String.codepoints()
+
+    # Initialize the game state utilizing the random word
+    %GameState{letters: _word} = struct(%GameState{}, letters: word)
```

Comment:
I'm not sure what you were trying to do here. `struct(%GameState{}, letters: word)` is the same as saying `%GameState{letters: word}`